content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md

```diff
@@ -0,0 +1,23 @@
+---
+weight: 60
+title: Release 10.15.0
+layout: redirect
+---
+
+Cumulocity IoT Edge Release 10.15, includes the following improvements, fixes and known issues.
+
+Download the Cumulocity IoT Edge version 10.15 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+### Removal of Telnet support
+
+Starting with Cumulocity IoT Edge Release 10.15, the support for the Telnet protocol has been removed.
+
+This change will affect you if you are trying to connect to the Edge appliance through remote access from the Cumulocity IoT tenant.
```

Comment:
- [ ] Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the {{< product-c8y-iot >}} tenant using Telnet protocol has been removed. {{< company-sag >}} recommends you to use the SSH protocol instead.

Reply:
Done